components/content/List/List.stories.ts

```diff
@@ -0,0 +1,147 @@
+import { StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+
+import '../Icon/Icon';
+import './List';
+
+export default {
+  component: 'diamond-list',
+  argTypes: {
+    variant: {
+      control: {
+        type: 'radio',
+      },
+      options: ['unstyled', 'icon'],
+    },
+    spacing: {
+      control: {
+        type: 'select',
+      },
+      options: [
+        'none',
+        'xs',
+        'sm',
+        'md',
+        'lg',
+        'xl',
+        'fluid-sm',
+        'fluid',
+        'fluid-lg',
+      ],
+    },
+  },
+};
+
+export const List: StoryObj = {
+  render: (args) => html`
+    <diamond-list variant="${args.variant}" spacing="${args.spacing}">
+      <ul>
+        <li>Item 1</li>
+        <li>Item 2</li>
+        <li>Item 3</li>
+      </ul>
+    </diamond-list>
+  `,
+};
+
+List.args = {
+  spacing: 'none',
+};
+
+export const Unstyled: StoryObj = {
+  render: (args) => html`
+    <diamond-list variant="${args.variant}" spacing="${args.spacing}">
+      <ul>
+        <li>Item 1</li>
+        <li>Item 2</li>
+        <li>Item 3</li>
+      </ul>
+    </diamond-list>
+  `,
+};
+
+Unstyled.args = {
+  variant: 'unstyled',
+};
+
+Unstyled.parameters = {
+  docs: {
+    description: {
+      story:
+        "Removes the list styles. This is useful for semantic lists that we don't want to see visually.",
```

Comment:
Worth adding a warning about safari in here https://developer.mozilla.org/en-US/docs/Web/CSS/list-style#accessibility_concerns

Reply:
Whoa, TIL!